### PR TITLE
[CHF-552] Implementation of Health Check for Aeon Siren.

### PR DESCRIPTION
**Pull request by skt123**
Added Health Check for Aeon Siren with checkinterval of 32 mins.

### CODE REVIEW

**Comment by skt123** [comment]
@jackchi @ShunmugaSundar 
Please review and merge.

**Comment by jackchi** [comment]
Let's quickly add a .README. 
@skt123  Does the `ping` generate an event? If so, what is it?

**Comment by skt123** [comment]
@jackchi Apart from the ping() event itself, no others events are being generated from ping. 
Ping just calls `zwave.basicV1.basicGet()` function to get device status as there is no `refresh()` method.

**Comment by skt123** [comment]
@jackchi 
Added Readme. Please check

File: devicetypes/smartthings/aeon-siren.src/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Aeon Siren
+
+Cloud Execution
```
Comment (jackchi) [inline devicetypes/smartthings/aeon-siren.src/README.md:3]: ZWave Execution?

Comment (skt123) [reply devicetypes/smartthings/aeon-siren.src/README.md:3]: That refers to the location of the code execution which in this case is **cloud** and not **local**.

**Pull request merged**